lib/telegraf/telegraf.conf.erb

```diff
@@ -553,14 +553,19 @@
 #   ]
 
 
+<% (config_file['mongodb'] || []).each do |server_url| %>
```

Comment:
Have you seen [this file](https://github.com/Instrumental/instrumentald/blob/master/lib/instrumentald/server_controller.rb#L190-L200)?